src/sortStudents.ts

```diff
@@ -1,16 +1,59 @@
-
+// not my solution
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'averGrade'
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order:SortOrder,
+): Student[] {
+  const sorted: Student[] = [...students];
 
-export function sortStudents(students, sortBy, order) {
-  // write your function
+  switch (sortBy) {
+    case SortType.Name:
+    case SortType.Surname:
+      return order === 'asc'
+        ? sorted.sort((a: Student, b: Student) => a[sortBy]
+          .localeCompare(b[sortBy]))
+        : sorted.sort((a: Student, b: Student) => b[sortBy]
+          .localeCompare(a[sortBy]));
+    case SortType.Age:
+    case SortType.Married:
+      return order === 'asc'
+        ? sorted.sort((a: Student, b: Student) => +a[sortBy] - +b[sortBy])
+        : sorted.sort((a: Student, b: Student) => +b[sortBy] - +a[sortBy]);
+    case SortType.AverageGrade:
```

Comment:
Add a blank line between cases
```suggestion
        : sorted.sort((a: Student, b: Student) => +b[sortBy] - +a[sortBy]);
        
    case SortType.AverageGrade:
```